### PR TITLE
[16.0][MIG] fieldservice_skill: Migration to version 16.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT43609

@pilarvargas-tecnativa  @chienandalu  please review!

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]

File: fieldservice_skill/views/fsm_person.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <field name="model">fsm.person</field>
+        <field name="inherit_id" ref="fieldservice.fsm_person_form" />
+        <field name="arch" type="xml">
+            <page name="contact" position="before">
+                <page string="Skills">
+                    <div class="row">
+                        <div
+                            class="o_hr_skills_group o_group_skills col-lg-5 d-flex flex-column"
+                        >
+                            <separator string="Skills" />
```
Comment (pilarvargas-tecnativa) [inline fieldservice_skill/views/fsm_person.xml:15]: In the fsm.person form the appearance of the "skill" tab is modified, although it does not affect the functioning, it looks a bit ugly without spacing.

V15:
image

V16:
image

Check other forms of the same style to see if there are any classes that have changed.

**Review by pilarvargas-tecnativa — APPROVED** [review]
thanks :)

File: fieldservice_skill/views/fsm_person.xml (hunk 1 above)
Comment (stefan-tecnativa) [reply fieldservice_skill/views/fsm_person.xml:15]: I think you didn't bring my last changes, i think it is displayed correctly right now. Check anyway if you see it as I do. 

imagen

**Comment by chienandalu** [comment]
Ready to merge

**Pull request merged**